### ISSUE TITLE
Tooltips must be dismissable, hoverable and available to mouse users as well  

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#27]
URL: https://hcdev.openplus.ca/en/user/login
User Agent: Chrome
Language: both

**Issue Description:** Tooltips must be dismissable, hoverable and available to mouse users as well  
**Issue 1:** There is no mechanism provided to dismiss the tooltip 
1. Using mouse hover over the Username field. Tooltip appears
2. Using keyboard press "escape"
3. Tooltip remains displayed. 
**Expected result Issue 1:** Provide a mechanism to easily dismiss the additional content, such as by pressing Escape.

**Issue 2:** Tooltip must be hoverable.
1. Using mouse hover over the Username field. Tooltip appears
2. Using mouse pointer attempt to hover over the tooltip. Tooltip disappears.
**Expected result Issue 2:** Tooltip should be hoverable itself

**Issue 3:** Tooltip must be available to mouse users as well
1. Tab to Username input. No tooltip appears
**Expected result issue 3**: information available for mouse users should be equally available to those using keyboard only.

Success criterion: https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

for these two tooltips there really is no added benefit to have these as they do not provide additional purpose to the fields. You may opt to remove these.

**Comment by davidgotlieb** [comment]
﻿no longer an issue, tooltips have been disabled as per mmiron-HC recommendation at ﻿https://github.com/mmiron-HC/WST-RSAMS/issues/69

**Comment by ndolter** [comment]
Validated in both French and English that tooltips are no longer available.
